ambry-named-mysql/src/main/java/com/github/ambry/named/NamedBlobState.java

```diff
@@ -0,0 +1,6 @@
+package com.github.ambry.named;
+
+public enum NamedBlobState {
+  WIP,
```

Comment:
nit: IN_PROGRESS, COMPLETE

Reply:
I think WIP is a well known abbr. for 'Work In Progress', so readability-wise it's ok. And it can save some storage/bits so :)